lib/ex_aws/dynamo.ex

```diff
@@ -268,18 +281,43 @@ defmodule ExAws.Dynamo do
         local_indexes,
         billing_mode \\ @default_billing_mode
       ) do
+    create_table(name, key_schema, key_definitions,
+      read_capacity: read_capacity,
+      write_capacity: write_capacity,
+      global_indexes: global_indexes,
+      local_indexes: local_indexes,
+      billing_mode: billing_mode
+    )
+  end
+
+  @doc """
+  Create table with arbitrary options. This provides a superset of the functionality of other
+  `create_table` variants, allowing the specification of any options as per the `t:create_table_opts/0`
+  type. If no options are specified, defaults will be used as follows:
+
+  * `billing_mode`: `#{inspect(@default_billing_mode)}`
+  * `read_capacity`: `#{@default_read_capacity}`
+  * `write_capacity`: `#{@default_write_capacity}`
+  * `streaming_enabled`: `false`
```

Comment:
And maybe some test coverage could be added around the `false`/`nil` cases?

Reply:
In my mind those values are the default *behaviour*, rather than the default values we send per se. I agree that that's a potentially unclear distinction, but I'm not sure what the answer is to clarifying it. The DDB docs are clear that not setting the value is the same as `false`, though, so implicitly that test is already kind of there, with the exception of the explicit false case. I'll add a test for that.

Reply:
Yeah I'm with you. But to my other point, is `:streaming_enabled` a typo, should that be `:stream_enabled`?